soccer/CMakeLists.txt

```diff
@@ -23,7 +23,7 @@ set(ROBOCUP_LIB_SRC
     "motion/TrapezoidalMotion.cpp"
     "NewRefereeModule.cpp"
     "planning/CompositePath.cpp"
-    "planning/DirectTargetPathPlanner.cpp"
+    #"planning/DirectTargetPathPlanner.cpp"
```

Comment:
Don't we want to keep DirectTargetPathPlanner for now until we find something better?  It'd be best to avoid degrading goalie performance in the meantime.